### ISSUE TITLE
Possible to support aspell directly?

### ISSUE THREAD

**Issue by ideasman42** [tlikonen/wcheck-mode#15]
This is an attempt to use aspell instead of enchant, it doesn't work and no errors are reported, could docs show an example for aspell since this is a popular spell checker?

```lisp
    (setq wcheck-language-data
      '
      (
        ("American English"
          (program . "/usr/bin/aspell")
          (args "-d" "en_US" "list")
          (action-program . "/usr/bin/aspell")
          (action-args "-a" "-d" "en_US" "list")
          (action-parser . wcheck-parser-ispell-suggestions)
          )
        )
      )
```

**Comment by vsemyonoff** [comment]
You need to add `(connection . pty)` to use it. The same with Hunspell.
